pkg/secret/secret.go

```diff
@@ -33,8 +33,9 @@ Please delete the component and recreate it using 'odo create'`, componentName)
 		if len(secrets) == 1 {
 			return secrets[0].Name, nil
 		}
-		return "", fmt.Errorf(`Multiple secrets exist for component %s. 
-Please select one of the following ports: '%s'`, componentName, strings.Join(availablePorts(secrets), ","))
+		return "", fmt.Errorf("Multiple secrets exist for component %s. "+
+			"Please select one of the following ports: '%s' "+
```

Comment:
Maybe something like
```
Please rerun the command by supplying the --port option with one of the following ports : `%s`
```
???

Reply:
Well, I just followed what was added in the open issue :)

Reply:
That was just a suggestion :)

Reply:
I don't mind either way of course :P

Reply:
Ofcourse the decision is upto you :D

Reply:
Let's go with this for the time being and if anyone complains we can always change it again

Reply:
ok :+1: